### ISSUE TITLE
[Fixed in GH-6052] Safari Firefox: Missing Visual Focus on Checkout: Choose Shipping Address

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4323]
Even though the buttons **Add New Address** and **Back to Cart** have visual focus this is not perceivable enough for accessibility purposes, it should be treated as missing visual focus.

Users with disabilities should be able to select or change addresses using the keyboard, currently the option **Ship to this address** is not keyboard accessible, therefore not receiving focus, thus no visual focus.

Buttons also missing visual focus

![Screen Shot 2019-08-29 at 4 19 28 PM](https://user-images.githubusercontent.com/15113219/63982172-9b879980-ca8f-11e9-84a7-b748fa51c2c9.png)

**Comment by Zeyber** [comment]
To be closed by #4993

**Comment by KateChuen** [comment]
Chrome OK

Missing keyboard accessibility and visual focus on:
- Add New Address button on Safari
- Back & Continue buttons on Safari
- Ship to this address button on Firefox

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-30 at 3 45 12 PM](https://user-images.githubusercontent.com/15113219/73488951-c1cf9f00-4377-11ea-800e-bcc90e09c97d.png)
![Screen Shot 2020-01-30 at 3 46 30 PM](https://user-images.githubusercontent.com/15113219/73488952-c1cf9f00-4377-11ea-8759-f3e3635b920d.png)
![Screen Shot 2020-01-30 at 3 46 45 PM](https://user-images.githubusercontent.com/15113219/73488953-c1cf9f00-4377-11ea-9d05-839058ce5a96.png)
![Screen Shot 2020-01-30 at 3 46 39 PM](https://user-images.githubusercontent.com/15113219/73488954-c1cf9f00-4377-11ea-8950-daf27d7f2735.png)

**Comment by KateChuen** [comment]
Tested on dev3.
chrome ✅
firefox ✅
safari ✅
